screens/ForgotPassword.screen.jsx

```diff
@@ -57,43 +55,32 @@ export const ForgotPasswordScreen = () => {
 
   const onSubmit = async () => {
```

Comment:
error modal ma się pokazywać tylko wtedy jeżeli leci błąd inny niż 400 lub 404

Reply:
musisz to wyłapać w checEmailValidation i tam wsadzić w else setShowErrorModal na true i wywalić go z catcha

Reply:
http://localhost:3005/api/#/auth/FindMeAuthController_sendPasswordResetLinkEmail na dole masz sprawdzanie jakie odpowiedzi lecą na request